pkg/volume/flexvolume/api/objects.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+)
+
+// Probe plugin
+
+type FlexVolumeDriverCapabilities struct {
+	AccessMode          api.PersistentVolumeAccessMode
+	DynamicProvisioning bool
+	Attachment          bool
+	LocalAttachment     bool // When Attachement is true, issue the attach/detach calls from the target host, not the controller.
+	CustomMount         bool // This driver has a custom mount/unmount logic. Mount/Unmount must be called.
```

Comment:
As the comment says :-) In more detail, we have default mount/unmount logic in the kubelet through the Mounter interface, and we need to know if the driver wants to implement mount his own way or not. The goal is to allow very simple drivers (for instance just attach/detach a volume on the host).